Wikipedia/View Controllers/WebView/WebViewController.m

```diff
@@ -341,17 +341,29 @@ - (CGFloat)clientBoundingRectVerticalOffset {
 }
 
 - (void)scrollToFragment:(NSString*)fragment {
-    if (fragment.length == 0) {
-        // No section so scroll to top. (Used when "Introduction" is selected.)
-        [self.webView.scrollView scrollRectToVisible:CGRectMake(0, 1, 1, 1) animated:NO];
+// Temp fix! See note below about bug with animated scroll approach...
+    if (fragment) {
+        NSString* jumpToThis = [fragment copy];
+        [[self.webView wmf_javascriptContext][@"scrollToFragment"] performSelector:@selector(callWithArguments:) withObject:@[jumpToThis] afterDelay:0.1];
```

Comment:
Short term solution is just to use this, which works. We would loose the scrolling animation, but could address that later when we have time...